site/troubleshooting.md

```diff
@@ -162,8 +166,8 @@ The columns are as follows:
       heartbeat
     * `established` - TCP connection and corresponding UDP path are up
  * Info - the failure reason for failed and retrying connections, or
-   the data transport method, remote peer name and nickname for
-   pending and established connections
+   the data transport method, security level, remote peer name and
+   nickname for pending and established connections
```

Comment:
> `plaintext` and `encrypted` were chosen as the two values because they are the same length

We don't make any reference "plaintext" for unencrypted weaves in the docs. And it doesn't sound right anyway, since we are not transmitting "text".

> I'd like the output to contain a positive indication that things are going in the plain

I appreciate that. But for those deployments running w/o a password - no doubt the majority - it is just noise.

> It's deliberately separate from the overlay type so we don't have to tinker with the status output when fastdp supports encryption

There is no tinkering required in what I proposed - the rule is "append `(encrypted)` when the connection is encrypted".